src/plugin.rs

```diff
@@ -900,6 +916,37 @@ impl Host for HostCallback {
             0.0,
         );
     }
+
+    /// Request time information from Host.
+    ///
+    /// The mask parameter is composed of the same flags which will be found in the `flags` field of `TimeInfo` when returned.
+    /// That is, if you want the host's tempo, the parameter passed to `get_time_info()` should have the `TEMPO_VALID` flag set.
+    /// This request and delivery system is important, as a request like this may cause
+    /// significant calculations at the application's end, which may take a lot of our precious time.
+    /// This obviously means you should only set those flags that are required to get the information you need.
+    ///
+    /// Also please be aware that requesting information does not necessarily mean that that information is provided in return.
+    /// Check the flags field in the `TimeInfo` structure to see if your request was actually met.
+    fn get_time_info(&self, mask: i32) -> Option<TimeInfo> {
+        if self.callback.is_none() {
+            return None
```

Comment:
Unnecessary check. This check is already done in `HostCallback::callback()`:
`let callback = self.callback.unwrap_or_else(|| panic!("Host not yet initialized."));`